pages/feed/index.tsx

```diff
@@ -75,16 +77,16 @@ const Feeds: WithPageLayout<HighlightSSRProps> = (props: HighlightSSRProps) => {
       router.push(`/feed?repo=${selectedRepo}`);
       setPage(1);
     }
-    if (highlightId) {
+    if (id) {
```

Comment:
This `id` does not change even after you pushed to `/feed`. Maybe it's because of the SSR :thinking: 

Thoughts @brandonroberts?

Reply:
That seems correct to me

Reply:
Alright, thanks